website/content/api-docs/system/internal-ui-default-auth-methods.mdx

```diff
@@ -0,0 +1,52 @@
+---
+layout: api
+page_title: /sys/internal/ui/default-auth-methods - HTTP API
+description: The '/sys/internal/ui/default-auth-methods' endpoint returns any configured or inherited default and backup auth methods for a given namespace.
+---
+
+# `/sys/internal/ui/default-auth-methods`
+
+@include 'alerts/enterprise-only.mdx'
+
+The `/sys/internal/ui/default-auth-methods` endpoint is used to return explicitly set or inherited default and backup auth method types for a given namespace.
+
+The namespace is derived from the request itself in the `X-Vault-Namespace` header.
+
+This should only be used internally by the UI. Due to the nature of its
+intended usage, there is no guarantee on backwards compatibility for this endpoint.
```

Comment:
```suggestion
Vault derives the approprate namespace from the `X-Vault-Namespace` request
header. `/sys/internal/ui/default-auth-methods` is not intended for general use
and makes no guarantee of backwards compatibility over time.
```

Reply:
I'm not convinced it's a dire enough situation to justify a warning box tbh

If we really feel we need a warning box, we should create a standardized alert (like the enterprise-only and restricted partials) as a partial that we can apply to all internal endpoints.